include/openthread/border_agent_txt_data.h

```diff
@@ -109,13 +109,15 @@ typedef enum otBorderAgentThreadRole
  */
 typedef struct otBorderAgentStateBitmap
 {
-    otBorderAgentConnMode      mConnMode;       ///< Connection Mode.
-    otBorderAgentThreadIfState mThreadIfState;  ///< Thread Interface Status.
-    otBorderAgentAvailability  mAvailability;   ///< Availability
-    otBorderAgentThreadRole    mThreadRole;     ///< Thread Role.
-    bool                       mBbrIsActive;    ///< Backbone Router function is active.
-    bool                       mBbrIsPrimary;   ///< Device is the Primary Backbone Router.
-    bool                       mEpskcSupported; ///< ePSKc Mode is supported.
+    otBorderAgentConnMode      mConnMode;                 ///< Connection Mode.
+    otBorderAgentThreadIfState mThreadIfState;            ///< Thread Interface Status.
+    otBorderAgentAvailability  mAvailability;             ///< Availability
+    otBorderAgentThreadRole    mThreadRole;               ///< Thread Role.
+    bool                       mBbrIsActive;              ///< Backbone Router function is active.
+    bool                       mBbrIsPrimary;             ///< Device is the Primary Backbone Router.
+    bool                       mEpskcSupported;           ///< ePSKc Mode is supported.
+    bool                       mMultiAilDetectionEnabled; ///< Multi-AIL detection is enabled.
```

Comment:
nit: may follow existing naming suffix rule as `mEpskcSupported`

```suggestion
    bool                       mMultiAilDetectionSupported; ///< Multi-AIL detection is enabled/supported.
```